### PR TITLE
define model

### PR DESCRIPTION
**Pull request by huuphuc93**

### CODE REVIEW

File: app/models/film.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Film < ApplicationRecord
+  has_many :post_reviews :ratings :categorys :watching_time
```
Comment (chuanhtuan) [inline app/models/film.rb]: mỗi has_many cho 1 dòng

File: app/assets/stylesheets/pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Pages controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/pages.scss]: remove this file

File: app/helpers/pages_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (dieunb) [inline app/helpers/pages_helper.rb]: remove this file

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :User
```
Comment (dieunb) [inline app/models/comment.rb]:
```
+  belongs_to :user
 +  belongs_to :post_review
```

**Pull request merged**